src/Test/Proxy/NginxProxy.php

```diff
@@ -15,7 +15,7 @@ class NginxProxy extends Abstractproxy
 {
     protected $binary = 'nginx';
     protected $configFile;
-    protected $port = 8080;
+    protected $port = 8088;
```

Comment:
Well. Actually that value is not used. The port is set here https://github.com/hpatoio/FOSHttpCache/blob/master/src/Test/NginxTestCase.php#L81 and as you can see is always overwritten.

I'll open another PR with some ideas about this.